packages/grafana-e2e/src/flows/addDashboard.ts

```diff
@@ -185,7 +186,7 @@ const addVariable = (config: PartialAddVariableConfig, isFirst: boolean): AddVar
     e2e.pages.Dashboard.Settings.Variables.Edit.QueryVariable.queryOptionsDataSourceSelect()
       .should('be.visible')
       .within(() => {
-        e2e.components.Select.input().should('be.visible').type(`${dataSource}{enter}`);
+        e2e.components.DataSourcePicker.inputV2().type(`${dataSource}{enter}`);
```

Comment:
hii for my own knowledge, do you know what this inputV2 thing is about? I sort of dropped away from e2e testing work and feel out of the loop haha

Reply:
I *think* this may be something to do with moving from `aria-labels` to `data-testid` (just from looking at the selectors). But someone from @grafana/user-essentials would have to confirm 🙃

Reply:
I *think* this may be something to do with moving from `aria-labels` to `data-testid` (just from looking at the selectors). But someone from @grafana/user-essentials would have to confirm 🙃